.dependabot/config.yml

```diff
@@ -862,3 +862,19 @@ update_configs:
         dependency_name: demisto/*
         update_type: all
 
+  - package_manager: docker
+    directory: /sane-pdf-reports
```

Comment:
whats that?

Reply:
@glicht will know better, it is something related to adding a new docker image to the repo.

Reply:
whats in `/sane-pdf-reports` ? i thought you use `/docker/sane-pdf-reports`

Reply:
It's automatic, I guess @glicht will know (maybe because I ran it from the directory and not the one above)